src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -30,6 +31,6 @@ void npe2(String a, String b) {
 
  @Override
  public boolean equals(Object obj) {
-  return true;
+  return false;
```

Comment:
Reporter: FINDBUGS
Rule: EQ_ALWAYS_FALSE
Severity: ERROR
File: se/bjurr/violations/lib/example/MyClass.java L34
 Source: se.bjurr.violations.lib.example.MyClass


equals method always returns false


  <p> This class defines an equals method that always returns false. This means that an object is not equal to itself, and it is impossible to create useful Maps or Sets of this class. More fundamentally, it means
that equals is not reflexive, one of the requirements of the equals method.</p>
<p>The likely intended semantics are object identity: that an object is equal to itself. This is the behavior inherited from class <code>Object</code>. If you need to override an equals inherited from a different
superclass, you can use use:</p>
<pre>
public boolean equals(Object o) { return this == o; }
</pre>


<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>